### PR TITLE
enable use of markdown

### PR DESCRIPTION
**Pull request by heather999**
Addressing #3 
Enabled the use of Markdown in the jinja2 templates of the portal.
The result can be viewed on the dev site: http://lb.desc-web-dev.production.svc.spin.nersc.org/
How this works:
* All the markdown files from https://github.com/LSSTDESC/DC2-Public-Release/tree/main/docs have been copied to this repo under `web/portal/templates/doc`.  Flask has some expectations about the location of files and for now, this was the easiest place to put these. Anyone who desires to update the content can edit the markdown directly.
* There are similar Flask expectations for the location of images.  The directions for downloading data using Globus includes images which are stored with the rest of the static image in `web/portal/static/img`.  These are referenced in the corresponding `download.md` file and are successfully rendered.
* There are new templates for every markdown page located in the `web/portal/templates` directory, with the exception of `getting_started.md` and `resources.md`.  Those templates include the corresponding markdown file via a block that looks like: 
```
   {% markdown %}
   {%include "doc/install_gcr.md"%}
   {% endmarkdown %}
```
* The `getting_started.md` and `resources.md` files are used to populate those sections on the home page via the `web/portal/templates/home.jinja2` template. 
* The routes are then defined in `web/portal/views.py`.  

Note that so far bullet points are not rendered using jinja_markdown, not sure why, but is not a showstopper.

The plan is to put this up for a quick review and then merge to main and push to production.

### CODE REVIEW

**Comment by heather999** [comment]
Yes, that sounds good.  Let me try it and see what happens

**Comment by JoanneBogart** [comment]
Is there a way to automatically generate the per-doc-page templates?

**Comment by heather999** [comment]
Yes, that's exactly when Yao suggested.  Now implemented and is working.

File: web/portal/templates/home.jinja2 [hunk 1]
```diff
@@ -76,17 +75,24 @@
             <!--First row-->
             <div class="row pb-4">
               <div class="col-10">
-                <h5 class="feature-title">Getting Started</h5>
-                <p class="grey-text">Overview and detailed instructions.</p>
+                {% markdown %}
+                {%include "doc/getting_started.md"%}
```
Comment (yymao) [inline web/portal/templates/home.jinja2:79]:
```suggestion
                {% include "doc/getting_started.md" %}
```

File: web/portal/templates/home.jinja2 [hunk 2]
```diff
@@ -76,17 +75,24 @@
...
+
+          <!--Grid column-->
+          <div class="col-lg-6 col-md-12 px-4">
             <!--Second row-->
             <div class="row pb-4">
               <div class="col-10">
-                <h5 class="feature-title">Resources</h5>
-                <p class="grey-text">Insert links to DESC Note & DC2 Paper.</p>
+                {% markdown %}
+                {%include "doc/resources.md"%}
```
Comment (yymao) [inline web/portal/templates/home.jinja2:94]:
```suggestion
                {% include "doc/resources.md" %}
```

File: web/portal/templates/doc_template.jinja2 [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{%extends "base.jinja2"%}
```
Comment (yymao) [inline web/portal/templates/doc_template.jinja2:1]:
```suggestion
{% extends "base.jinja2" %}
```

File: web/portal/templates/doc_template.jinja2 [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+{%block body%}
```
Comment (yymao) [inline web/portal/templates/doc_template.jinja2:3]:
```suggestion
{% block body %}
```

File: web/portal/templates/doc_template.jinja2 [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+  {%include "header.jinja2"%}
```
Comment (yymao) [inline web/portal/templates/doc_template.jinja2:5]:
```suggestion
  {% include "header.jinja2" %}
```

File: web/portal/templates/doc_template.jinja2 [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+<div class="container">
+
+   {% markdown %}
+   {% include "doc/" + doc_name + ".md" %}
+   {% endmarkdown %}
+
+</div>
+<!-- container -->
+
+{%endblock%}
```
Comment (yymao) [inline web/portal/templates/doc_template.jinja2:16]:
```suggestion
{% endblock %}
```

**Review by yymao — APPROVED** [review]
LGTM. I've very annoying added some code style comments. It's ok to ignore me.

**Pull request merged**